src/steps/account/account-create.ts

```diff
@@ -11,14 +11,25 @@ export class CreateAccount extends BaseStep implements StepInterface {
     type: FieldDefinition.Type.MAP,
     description: 'A map of field names to field values',
   }];
+  protected expectedRecords: ExpectedRecord[] = [{
```

Comment:
- [ ] Relatedly, would want to see the same create/modified dates on the account object too:

```
[{
  field: 'CreatedDate',
  type: FieldDefinition.Type.DATETIME,
  description: "Account's Created Date",
}, {
  field: 'LastModifiedDate',
  type: FieldDefinition.Type.DATETIME,
  description: "Account's Last Modified Date",
}],
```